### DISCUSSION TITLE
How to use DELETE request method

### DISCUSSION THREAD

**Discussion by jamiehaywood** [johnlindquist/kit#724] [Q&A]
Awesome application, really enjoying using it.

I'm trying to send a DELETE request. Unfortunately I can't override with `{ method: 'DELETE' }`.

Any suggestions?

Cheers

**Comment by johnlindquist** [comment]
`fetch` is availabe:

```js
let url = `foo.com`

await fetch(url,{
  method:'DELETE'
})
```

I've honestly held off on adding `delete` because I have no idea what to name it 😬 Maybe `del`?
